webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvSubject.java

```diff
@@ -0,0 +1,19 @@
+package org.opentestsystem.rdw.admin.model;
+
+/**
+ * This class represents the Group subjects allowed during CSV Group uploads.
+ */
+public enum CsvSubject {
```

Comment:
I believe that we're assuming that it's as if All is specified somewhere, but that's the responsibility of the group-processor @chdietz